### PR TITLE
Revised homepage / pricing / demo

### PR DESCRIPTION
**Pull request by mgamini**
tenor-261712423

View this in action at [http://hab-wip.surge.sh](http://hab-wip.surge.sh)

Definitely not perfect. Always more polish to do!

This affects the following issues:

- [#3317](https://github.com/habitat-sh/habitat/issues/3317)
- [#3316](https://github.com/habitat-sh/habitat/issues/3316)
- [#3341](https://github.com/habitat-sh/habitat/issues/3341)

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by raskchanky** [comment]
@mgamini I'd recommend squashing the commits ;)

**Comment by reset** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/282384195) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/282384195) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/282384195) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/282384195).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/282384195) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by tashimi** [comment]
@mgamini This needs to get rebased so we can deploy it to acceptance :)

**Comment by reset** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282462286).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/282462286).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/demo/steps/1.html.slim [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        .columns.large-12
+          h1.h4="Step 1: Fork the sample app on Github"
+        .columns.large-6
+          p
+            ' For this demo, you will be using a sample Node.js application. To get the application, you will need to be signed in to
+            a.link-external target="_bank" href="https://github.com" github.com
+            '  so that you can fork the repo.
+          p
+            ' Open the sample repo at
+            a.link-external target="_bank" href="https://github.com/habitat-sh/habitat-demo-node" https://github.com/habitat-sh/habitat-demo-node
```
Comment (tashimi) [inline www/source/demo/steps/1.html.slim]: https://github.com/habitat-sh/sample-node-app

Comment (tashimi) [reply www/source/demo/steps/1.html.slim]: ^^ that is the correct link

File: www/source/demo/steps/6.html.slim [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            ' Switch the
+            em Publish to Docker Hub
+            '  option to
+            strong ON
+            '  - the Docker Hub account you added in the previous step should appear and be selected.
+          p
+            ' Finally, select a target Docker Hub repo
+            span.asterisk.sup="*"
+            '  from the dropdown. You can leave the
+            em Docker Tag
```
Comment (tashimi) [inline www/source/demo/steps/6.html.slim]: this text runs into the next word, maybe add another space

File: www/source/demo/steps/7.html.slim [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          p You've set up automated builds and Docker container publishing. Now, let's watch Habitat Builder at work!
+          p
+            ' From the My Origins Packages view, you'll see a message when a new build job kicks off (see image below). Once the message appears, click on the
+            strong View build output
+            '  link to see streaming logs as your job completes.
+          img src="/images/demo/screen-step-7a.png"
+        .columns.large-6
+          p
+            ' When the job is complete, click the
+            strong Published to
```
Comment (tashimi) [inline www/source/demo/steps/7.html.slim]: this to runs into the next word in the UI maybe add another space

File: www/source/demo/steps/9.html.slim [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            '  file in your forked GitHub repo (as we did earlier) by clicking the
+            span.icon-pencil pencil icon
+            ' . Change the value of
+            strong  pkg_version
+            '  from
+            em  1.0.1
+            '  to
+            em  1.0.2
+            ', then use the
+            strong  Commit changes
```
Comment (tashimi) [inline www/source/demo/steps/9.html.slim]: this changes runs into the next word

File: www/source/layouts/_footer.slim [hunk 5]
```diff
@@ -31,7 +41,7 @@ footer#main-footer class="#{layout_class}"
           .columns.medium-2.small-6
             ul.no-bullet
               li
-                h4 Build Service
+                h4.footer--subheader Build Service
```
Comment (tashimi) [inline www/source/layouts/_footer.slim]: Should this be Builder?

**Comment by tashimi** [comment]
We should wait to merge this until the functionality it's portraying is merged in Builder

**Comment by ryankeairns** [comment]
Notes of changes coming down the pike:

- we’ll need to add in the GitHub Apps activation dance (this may be an additional step)
- the Docker repo selection and tagging has changed from a select dropdown (so there will be screenshots and copy changes)
- the sign in screen will also be changing (so need to change that screenshot too)

**Comment by apriofrost** [comment]
Here is a doc that includes the feedback for the home/demo/pricing page (critical ones are marked with a 💟 )
https://docs.google.com/document/d/1HSMyzCEqWkT9e99SicWu4xZvipAofaQr9PJQ6brXSww/edit#

File: www/source/demo/steps/7.html.slim [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        a.demo-step--indicator.active href="/demo/steps/7"
+        a.demo-step--indicator href="/demo/steps/8"
+        a.demo-step--indicator href="/demo/steps/9"
+      .row
+        .columns.large-12
+          h1.h4="Step 7: It's Happening!"
+        .columns.large-6
+          p You've set up automated builds and Docker container publishing. Now, let's watch Habitat Builder at work!
+          p
+            ' From the My Origins Packages view, you'll see a message when a new build job kicks off (see image below). Once the message appears, click on the
```
Comment (apriofrost) [inline www/source/demo/steps/7.html.slim]: Missing a ">" between "My Origins" and "Packages"

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282912048).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282912048).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282912048).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282912048).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282912048).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/282912048).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/stylesheets/_pricing.scss [hunk 7]
```diff
@@ -0,0 +1,151 @@
+.pricing-row-wrapper {
+  position: relative;
+
+  @include breakpoint(medium) {
+    .background-waves-blue {
+      position: absolute;
+      background: url('/images/background-dark-blue.png') center top repeat-x;
```
Comment (apriofrost) [inline www/source/stylesheets/_pricing.scss]: The background image needs update, download new asset here: https://drive.google.com/open?id=0B6wx-y4RiRvoTlk5aGpXeDI3Mnc

File: www/source/stylesheets/_pricing.scss [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+    border-radius: 0px 0px $global-radius $global-radius;
+  }
+}
+
+.pricing--contact-us {
+  min-height: 373px;
+  width: 100%;
+  margin-top: rem-calc(-150);
+  position: relative;
+  background: url('/images/graphics/bg-mountains-green2.png') center 50px repeat-x transparent;
```
Comment (apriofrost) [inline www/source/stylesheets/_pricing.scss]: This background image needs update, download the new asset here: https://drive.google.com/open?id=0B6wx-y4RiRvoWHBpQ1pFYlZWUDg

Comment (tashimi) [reply www/source/stylesheets/_pricing.scss]: @apriofrost looks like we need these in *.png, not *.svg, or do you want me to change the format?

Comment (apriofrost) [reply www/source/stylesheets/_pricing.scss]: Yes. Change the format.

File: www/source/stylesheets/_components.scss [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+.mt-4 { margin-top: rem-calc($std-margin * 4); }
+
+.mb-0 { margin-bottom: 0px; }
+.mb-1 { margin-bottom: rem-calc($std-margin); }
+.mb-2 { margin-bottom: rem-calc($std-margin * 2); }
+.mb-3 { margin-bottom: rem-calc($std-margin * 3); }
+.mb-4 { margin-bottom: rem-calc($std-margin * 4); }
+
+.background-waves-green {
+  background: url('/images/background-waves-green.png') repeat-x center center;
```
Comment (apriofrost) [inline www/source/stylesheets/_components.scss]: This background image asset needs update, download new asset here: https://drive.google.com/open?id=0B6wx-y4RiRvoLWlTOUE1Tk82VW8

Comment (tashimi) [reply www/source/stylesheets/_components.scss]: done!

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282988074).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282988074).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282988074).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/282988074).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/282988074).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/282988074) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283014172).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283014172).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283014172).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283014172).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283014172).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/283014172).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283031033).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/283031033).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283233309).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/283233309).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/283233309) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

File: www/source/pricing/index.html.slim [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+        p.mb-0 Docker Hub only, more coming
+      .columns.large-12.pricing-box-row
+        p.h6.mb-0 Scaffoldings
+        p.mb-0 Ruby, Node, JVM, with more all the time
+      .columns.large-12.pricing-box-row.background-gray-light
+        p.h6.mb-0 Export your application to many formats
+        p.mb-0 Docker container, Application Container Image (ACI), AMI, Tar
+      .columns.large-12.pricing-box-row
+        p.h6.mb-0 Compatible with your favorite scheduler
+        p.mb-0 Kubernetes, CloudFoundry, Mesos DC/OS, Terraform, and more
```
Comment (reset) [inline www/source/pricing/index.html.slim]: Terraform is a provisioner and not a scheduler. Should we drop it or alter the header here?

**Comment by ryankeairns** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
Hey, @ryankeairns - you don't have permission to issue `deploy` on this pull request.

Nope

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283262416).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/283262416).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by ryankeairns** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283331682).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/283331682).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by reset** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283574687).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/283574687).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by ryankeairns** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/283751744).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/283751744).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/283751744) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by mgamini** [comment]
@ryankeairns @apriofrost @reset @cnunciato 
- Mobile styling
- Integrating feedback from feedback doc

**Comment by ryankeairns** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/285303457).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/285303457).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by ryankeairns** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/285329102).

**Pull request merged**